### PR TITLE
feat: allow airgapped installs

### PR DESCRIPTION
**Pull request by mnaser**

### CODE REVIEW

File: atmosphere/operator/controllers/cloud.py [hunk 1]
```diff
@@ -91,6 +91,15 @@ def create_fn(namespace: str, name: str, spec: dict, **_):
             ),
         )
 
+    flow.add(
+        tasks.ApplyNamespaceTask(
+            name=constants.NAMESPACE_OPENSTACK,
+            provides="openstack_namespace",
+        ),
```
Comment (mnaser) [inline atmosphere/operator/controllers/cloud.py]: no need for this, we already have `namespace` which is created earlier :)

File: atmosphere/operator/controllers/cloud.py [hunk 2]
```diff
@@ -91,6 +91,15 @@ def create_fn(namespace: str, name: str, spec: dict, **_):
...
+        tasks.ApplyHelmReleaseTask(
+            config=constants.HELM_RELEASE_INGRESS_NGINX,
+        ),
```
Comment (mnaser) [inline atmosphere/operator/controllers/cloud.py:97]: this creates a regression, it is not possible to "disable" ingress-nginx with this...

File: atmosphere/operator/controllers/cloud.py [hunk 3]
```diff
@@ -177,11 +186,16 @@ def create_fn(namespace: str, name: str, spec: dict, **_):
         flow.add(tasks.ApplyRabbitmqClusterTask("heat"))
 
     if spec["monitoring"]["enabled"]:
+        # TODO(oleks): Enable opsgenie
```
Comment (mnaser) [inline atmosphere/operator/controllers/cloud.py:185]: and make sure there's a way to do overrides too

File: atmosphere/operator/controllers/cloud.py (hunk 2 above)
Comment (mnaser) [reply atmosphere/operator/controllers/cloud.py:97]: and make sure there's a way to do overrides too

**Comment by okozachenko1203** [comment]
@mnaser i cannot resolve these conflicts because it doesn't update head

**Comment by mnaser** [comment]
this is already implemented via the `image_manifest` role:

https://github.com/vexxhost/atmosphere/blob/90f91b3dda9fed24bad292cf232cce9259d70635/roles/image_manifest/README.md#L4

**Pull request closed without merging**